CHANGELOG.md

```diff
@@ -142,6 +143,7 @@
 - `tailsamplingprocessor`: Add And sampling policy (#6910)
 - `coralogixexporter`: Add Coralogix Exporter (#7383)
 - `prometheusexecreceiver`: Add default value for `scrape_timeout` option (#7587)
+- `prometheusremotewriteexporter`: Write-Ahead Log support enabled (#7304)
```

Comment:
Two log entries, one in breaking changes and one in enhancements?

Reply:
Sigh...

Merge conflicts in the changelog are the worst!  They're both in `Enhancements` but this line is from ages ago.

```suggestion
```